Sources/TensorFlow/Layers/Core.swift

```diff
@@ -158,6 +165,7 @@ public struct Dense<Scalar: TensorFlowFloatingPoint>: Layer {
         self.weight = weight
```

Comment:
Should we add preconditions here to assert something like `weight.rank <= 3` and `bias.rank <= 2`?

Reply:
Done.